packages/lesswrong/server/repos/DatabaseMetadataRepo.ts

```diff
@@ -78,6 +81,56 @@ export default class DatabaseMetadataRepo extends AbstractRepo<"DatabaseMetadata
     return this.getGivingSeasonHearts(electionName);
   }
 
+  async getBannerEmojis(): Promise<BannerEmoji[]> {
```

Comment:
The giving season functions are still above as (I think) they're still used by LW, but they're sufficiently different that we need to just make some new ones - some cleanup is definitely needed here (dito for the graphql resolvers).